### PR TITLE
Adjust configuration files

### PR DESCRIPTION
**Pull request by qiaojialin**
1. Adjust the order of parameters in xx.properties for easy use
2. Change disk_space_warning_threshold from 5.0 to 0.05
3. Change default cn_internal_address/dn_internal_address/dn_rpc_address from 0.0.0.0 to 127.0.0.1
4. Rename all partition_region_xxx to config_node_xxx
5. Rename all xx_target_config_nodes to xx_target_config_node_list
6. Rename some remaining OneCopy to SimpleConsensus

After these changes, the system.properties needs to update for historical data upgrade.

### CODE REVIEW

File: confignode/src/assembly/resources/conf/iotdb-confignode.properties [hunk 1]
```diff
@@ -21,42 +21,33 @@
...
+cn_internal_address=127.0.0.1
 
-# Used for cluster internal RPC communication
+# Used for RPC communication inside cluster.
 # Datatype: int
 cn_internal_port=22277
 
-
-# Used for ConfigNodeGroup's ConsensusLayer interior communication
+# Used for communication among ConfigNodes inside cluster.
```
Comment (Beyyes) [inline confignode/src/assembly/resources/conf/iotdb-confignode.properties:33]:
```suggestion
# Used for consensus communication among ConfigNodes inside cluster.
```
is better?

File: confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConfig.java [hunk 2]
```diff
@@ -155,39 +155,39 @@ public class ConfigNodeConfig {
    */
   private long dataRegionRatisSnapshotTriggerThreshold = 400000L;
 
-  private long partitionRegionRatisSnapshotTriggerThreshold = 400000L;
-  private long partitionRegionOneCopySnapshotTriggerThreshold = 400000L;
+  private long configNodeRatisSnapshotTriggerThreshold = 400000L;
+  private long configNodeOneCopySnapshotTriggerThreshold = 400000L;
```
Comment (Beyyes) [inline confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConfig.java:159]: I will replace `OneCopy` to `Simple` next pr

File: server/src/main/java/org/apache/iotdb/db/service/thrift/impl/DataNodeInternalRPCServiceImpl.java [hunk 3]
```diff
@@ -1037,8 +1037,8 @@ private void sampleDiskLoad(TLoadSample loadSample) {
             .value();
 
     if (freeDisk != 0 && totalDisk != 0) {
-      double freeDiskRatio = (double) freeDisk * 100 / totalDisk;
-      loadSample.setDiskUsageRate(100.0 - freeDiskRatio);
+      double freeDiskRatio = (double) freeDisk / totalDisk;
+      loadSample.setDiskUsageRate(1 - freeDiskRatio);
```
Comment (Beyyes) [inline server/src/main/java/org/apache/iotdb/db/service/thrift/impl/DataNodeInternalRPCServiceImpl.java:1041]: 1.0?

File: node-commons/src/assembly/resources/conf/iotdb-common.properties [hunk 4]
```diff
@@ -15,77 +15,311 @@
...
+# 2. org.apache.iotdb.consensus.multileader.MultiLeaderConsensus
+# 3. org.apache.iotdb.consensus.ratis.RatisConsensus
+# Datatype: String
+# data_region_consensus_protocol_class=org.apache.iotdb.consensus.simple.SimpleConsensus
+
+####################
+### Partition (Load balancing) configuration
+####################
+
+# All parameters in PartitionSlot configuration is unmodifiable after ConfigNode starts for the first time.
```
Comment (Beyyes) [inline node-commons/src/assembly/resources/conf/iotdb-common.properties:61]:
```suggestion
# All parameters in Partition configuration is unmodifiable after ConfigNode starts for the first time.
```

File: node-commons/src/assembly/resources/conf/iotdb-common.properties [hunk 5]
```diff
@@ -15,77 +15,311 @@
...
+# series_partition_executor_class=org.apache.iotdb.commons.partition.executor.hash.BKDRHashExecutor
+
+# The maximum number of SchemaRegion expected to be managed by each DataNode.
+# Notice: Since each StorageGroup requires at least one SchemaRegion to manage its schema,
+# this parameter doesn't limit the number of SchemaRegions when there are too many StorageGroups.
+# Datatype: Double
+# schema_region_per_data_node=1.0
+
+# The maximum number of DataRegion expected to be managed by each processor.
+# Notice: Since each StorageGroup requires at least two DataRegions to manage its data,
```
Comment (Beyyes) [inline node-commons/src/assembly/resources/conf/iotdb-common.properties:85]: `at least two DataRegions to manage its data` is inappropriate?

File: node-commons/src/assembly/resources/conf/iotdb-common.properties [hunk 6]
```diff
@@ -15,77 +15,311 @@
...
+# this parameter doesn't limit the number of DataRegions when there are too many StorageGroups.
+# Datatype: Double
+# data_region_per_processor=0.5
+
+# Region allocate strategy
+# These allocate strategies are currently supported:
+# 1. GREEDY(Default, when region is allocated, always choose the dataNode that has bean allocated the least regions)
```
Comment (Beyyes) [inline node-commons/src/assembly/resources/conf/iotdb-common.properties:92]:
```suggestion
# 1. GREEDY(Default, when region is allocated, always choose the dataNode that has been allocated the minimum regions)
```

File: node-commons/src/assembly/resources/conf/iotdb-common.properties [hunk 7]
```diff
@@ -15,77 +15,311 @@
...
+# 2. COPY_SET(Random replication according to wight calculated from number of regions on all online dataNodes, suitable for large clusters)
```
Comment (Beyyes) [inline node-commons/src/assembly/resources/conf/iotdb-common.properties:93]:
```suggestion
# 2. COPY_SET(Random replication according to weight calculated from number of regions on all online dataNodes, suitable for large clusters)
```

**Review by HTHou — APPROVED** [review]

**Pull request merged**